recurly/errors.py

```diff
@@ -6,69 +6,143 @@
 import recurly
 
 
-class BadRequestError(recurly.ApiError):
+def error_from_status(status):
+    error_map = {
```

Comment:
Anyway we can make this a module-level constant since it shouldn't change over the course of the program? it should save some allocations.

Reply:
Ah yes, updated.